### PR TITLE
Issue/599

### LINKED ISSUE #30: simulator and sut delta times are swapped
sys\server\src\TxsServer.hs

Line 252:
```
cmdStop :: String -> IOS.IOS ()
cmdStop _  =  do
     World.closeSockets
     modus <- gets IOS.modus
     if  IOS.isSimuled modus
       then do [(nmsut,valsut)] <- IOS.getParams ["param_Sut_deltaTime"]
               [(nmsim,valsim)] <- IOS.getParams ["param_Sim_deltaTime"]
==>            IOS.setParams [("param_Sut_deltaTime",valsim)]
==>            IOS.setParams [("param_Sim_deltaTime",valsut)]
…
```
Shown lines swap delta time values of sim and sut.

### LINKED ISSUE #599: Sut/Sim delta time should be a parameter
param_Sut_deltaTime is implicit in World.putSocket.

SocketWorld.hs:
```
putSocket envs ActQui =
     let ( Just towChan, _,  _ )  = IOS.tow envs
         ( Just frowChan, _,  _ ) = IOS.frow envs
         deltaTime                = case Map.lookup "param_Sut_deltaTime" (IOS.params envs) of
                                      Nothing      -> 2000                -- default 2 sec
                                      Just (val,_) -> read val
         ioTime                   = case Map.lookup "param_Sut_ioTime" (IOS.params envs) of
                                      Nothing      -> 10                -- default 10 msec
                                      Just (val,_) -> read val
```

TxsServer.hs:
```
cmdSimulator ... = ...
lift $ TxsCore.txsSetSim (World.putSocket envs) (World.getSocket envs)
                         modeldef (Just mapperdef)
```

Related to #30 , #561 and #578

### PR DESCRIPTION
**Pull request by pjljvandelaar**
fixes #599
Resolves #30 :tada:

### CODE REVIEW

**Comment by keremispirli** [comment]
3 failed tests, all of them suggest that `Sut_deltaTime` is too low now.

File: test/sqatt/src/Sqatt.hs [hunk 1]
```diff
@@ -317,6 +317,7 @@ runTxsWithExample mLogDir ex delay = Concurrently $ do
     txsUIProc mUiLogDir imf port =
       Concurrently $ do
         eRes <- try $ Turtle.fold txsUIShell findExpectedMsg
+        sleep 5.0       -- take time to write results to log files (in case of error)
```
Comment (capitanbatata) [inline test/sqatt/src/Sqatt.hs]: We should remove this sleep in the future. I've created https://github.com/TorXakis/TorXakis/issues/601

**Comment by pjljvandelaar** [comment]
@keremispirli Do you like my new changes better?

**Comment by keremispirli** [comment]
Looks like it had to be even lower! :D

**Comment by keremispirli** [comment]
Wait, doesn't this also close #30, #561 and #578 ?

**Pull request merged**